project.yaml

```diff
@@ -938,23 +940,40 @@ Layer:
     <<: *extents
     Datasource:
       <<: *osm2pgsql
+      # This is one of the most complex layers, so it bears explaining in some detail
+      # It is necessary to
+      # - Have roads and railways in the same layer to get ordering right
+      # - Return two linestrings for ways which are both a road and railway
+      # - Compute z_order here, rather than rely on osm2pgsql z_order
       table: |-
```

Comment:
I don't think anything on aeroways belongs here as there aren't any aeroway requirements influencing how we need to write the query. The other points are about what requires the `UNION ALL` and the z_order `JOIN`